packages/styles/components/_c.checkbox.scss

```diff
@@ -12,6 +12,7 @@
 
     cursor: pointer;
     margin-left: magic-unit-rem(0.5, true);
+    color: $color-font-darker;
```

Comment:
According to the Figma mockups, I think that the right color is: `$color-font-darkest` that is the equivalent of `#000`